app/views/placeholder/_orgs.html.erb

```diff
@@ -6,13 +6,13 @@
     <div class="modal-dialog" role="document">
```

Comment:
Not a line changed in this PR but since we're leaning towards showing all the organizations if they haven't taken the quiz we should change line 1 of this ` <h3 class="matched"> Your matched organizations </h3> ` to say something like "All organizations you can potentially match with" -- or whatever the client thinks is an appropriate label -- when they haven't taken the quiz and then change it to Your matched orgs when they have taken it.

Reply:
Correct me if I'm wrong, but you have developed an alternative that does not involve the generate_orgs route at all, so this will be overwritten anyway and I won't bother making any changes for it here.